tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "outDir": "dist",
     "resolveJsonModule": true,
     "sourceMap": true,
-    "target": "ES2017",
+    "target": "ES2018",
```

Comment:
might want to bump this up to ES2020 where you'd get access to optional chaining. might even just go to ES2024.